.travis.yml

```diff
@@ -20,7 +20,7 @@ before_install:
   - pip install -U https://github.com/openwisp/openwisp-controller/tarball/master
   - pip install -U https://github.com/openwisp/openwisp-monitoring/tarball/master
   - pip install -U --no-deps https://github.com/openwisp/openwisp-notifications/tarball/master
-
+  - pip install -U https://github.com/openwisp/openwisp-users/tarball/master
```

Comment:
Simply adding `openwisp-users ~=0.4.0` to `requirements.txt` causes travis CI to fail with `AttributeError` exception saying `User object has no attribute organizations_managed`. But adding this line solves the problem. Don't know why this is happening

Reply:
It's because of this: https://github.com/openwisp/openwisp-network-topology/blob/master/requirements-test.txt#L4
Update it there as well and this line won't be necessary (please remove it).